tests/gis_tests/geoapp/tests.py

```diff
@@ -221,9 +222,22 @@ def test_empty_geometries(self):
             self.assertEqual(feature.geom.srid, g.srid)
 
 
+def mysql_disjoint_bug(func):
+    """
+    It's still unknown why the MySQL disjoint lookup gives different results
```

Comment:
How different is this? Is it something that should be documented?

Reply:
chop "still" and "in our tests"